test-scripts/run-anchor-tests.sh

```diff
@@ -1,10 +1,12 @@
 if [ "$1" != "--skip-build" ]
   then
     anchor build &&
-    cp target/idl/clearing_house.json sdk/src/idl/
+    cp target/idl/clearing_house.json sdk/src/idl/ && 
+    cd sdk && yarn build && cd ../
```

Comment:
can we add a flag to the script that forces the sdk to rebuild. i dont want this to build for me every time